app/mailers/producer_mailer.rb

```diff
@@ -0,0 +1,37 @@
+
+class ProducerMailer < Spree::BaseMailer
+
+  def order_cycle_report(producer, order_cycle)
+    @producer = producer
+    @coordinator = order_cycle.coordinator
+    @order_cycle = order_cycle
+
+    subject = "[#{Spree::Config[:site_name]}] Order cycle report"
+
+    # if @order_cycle.distributors.any?
+    #   first_producer = @order_cycle.distributors.first
+    #   @distribution_date = @order_cycle.pickup_time_for first_producer
+    #   subject += " for #{@distribution_date}" if @distribution_date.present?
+    # end
+
+    # TODO: what if producer handling orders into multiple order cycles?
```

Comment:
Is this a situation you envisage happening regularly? I can see the advantage in batching up these into a single email per producer, but I can also see it being tricky coordinating emails across several order cycles that close at different times.